docs/contributor/04-50-cluster-vs-namespaced-scoped.md

```diff
@@ -0,0 +1,65 @@
+# Pros and cons of cluster scoped and namespaced scoped module custom resource
+
+
+<table style="table-layout: fixed; width: 100%">
+    <tbody>
+        <tr>
+            <td colspan="2" align="center">Cluster scoped Custom Resource</td>
+            <td colspan="2" align="center">Namespaced scoped Custom Resource</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;" align="center">Pros</td>
+            <td style="width: 25%;" align="center">Cons</td>
+            <td style="width: 25%;" align="center">Pros</td>
+            <td style="width: 25%;" align="center">Cons</td>
+        </tr>
+        <tr>
+            <td>[UX] Resource is more visible to customer, and the module is clearly visible as installed.<br></br> This might also guarantee better visibility that this resource manages resources across the cluster, not only in a single namespace.</td>
+            <td>Different approach than other Kyma modules</td>
+            <td>Resource closer to manager</td>
+            <td>[UX] Module is less visible in Dashboard, as user needs to access Kyma system, before seeing the resource</td>
+        </tr>
+        <tr>
+            <td>Ability to use cross namespace owner references:<br></br> - Allows getting all managed resources by owner reference<br></br>- Allows easy to do force deletion of the module, even without a controller</td>
+            <td>[UX] Module ties to Kyma might not be as visible</td>
+            <td>[UX] Resource is a clear part of Kyma</td>
+            <td>Creation and modifying of resources in `kyma-system` is required, even in LM scenario</td>
+        </tr>
+        <tr>
+            <td>[UX] User does not need to create or edit resource in `kyma-system`, which should only be managed by Kyma team</td>
+            <td>Requires additional configuration to have it displayed in Busola under `kyma-system`, if we consider that to be the desired behaviour</td>
+            <td></td>
+            <td>Reconciliation for resources in different Namespaces is anyway needed, as they need to be put in `Warning` state</td>
+        </tr>
+        <tr>
+            <td>[UX] Configuring RBAC for creation/visibility of the module is easier (in a cluster that has users with different level of access, e.g. administrator, developer, user)<br></br>This allows a more fine grained approach to configuring ClusterRoles without the need to expose access to `kyma-system`</td>
+            <td></td>
+            <td></td>
+            <td>Owner references don't bring any value, as they would only allow access to `kyma-system` resources</td>
+        </tr>
+        <tr>
+            <td>Monitoring number of resources is easier, and doesn't need to have access to `kyma-system`</td>
```

Comment:
What resources are we referring to? I think we should add some context.

Reply:
clarify about it being module CR